src/pages/instances/actions/StartInstanceBtn.tsx

```diff
@@ -62,12 +62,13 @@ const StartInstanceBtn: FC<Props> = ({ instance }) => {
       aria-label={isLoading ? "Starting" : "Start"}
       title="Start"
     >
-      <i
+      <Icon
         className={
           isLoading
-            ? "p-icon--spinner u-animation--spin"
-            : "p-icon--start-instance"
+            ? "instance-state-icon u-animation--spin"
```

Comment:
Still a leftover of the `instance-state-icon`, which can be removed. We could use the `classnames` helper here maybe?

Reply:
Whoops, yes, it can be removed, and good idea about the `classnames` - this actually made me realise that we need exactly the same thing in the `ConfirmationButton`, because when I removed the `baseIcon` I forgot to apply the `u-animation--spin` class when loading, leaving it with a "still" spinner.